qube/create/templates/common_files/{{ cookiecutter.commonName }}/.github/workflows/nexus-publish.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build a package using Maven and then publish it to
+#  qbic-repo.qbic.uni-tuebingen.de packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Nexus Package
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - development
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+        settings-path: ${{ github.workspace }} # location for the settings.xml file
```

Comment:
You need raw statements here since cookiecutter interprets this as jinja2 syntax.

Reply:
{% raw %} settings-path: ${{ github.workspace }} # location for the settings.xml file {% endraw %}

Reply:
Ah, I am not used to Jinja yet, nice spot!

Reply:
Besides spotting it, the failing workflows basically scream this issue at me.

I've had own fair share of failing template creation due missing raw statements :)